devtools/create_engine/files/xyzzy.h

```diff
@@ -74,10 +74,10 @@ class XyzzyEngine : public Engine {
 		    (f == kSupportsReturnToLauncher);
 	};
 
-	bool canLoadGameStateCurrently() override {
+	bool canLoadGameStateCurrently(Common::U32String *msg = nullptr) override {
```

Comment:
I think it's better to avoid default values in the subclasses. You can have the default in the base class alone.

This way, when you explicitly call this function for a subclass (like when calling it for a parent), you won't miss passing the message.

Reply:
I do not quite understand, could you please explain a bit more? This is an override, superclass is called only when it was not overridden.

Reply:
I mean explicit calls from a subclass to its super.

Example:
```c++
struct Base {
  virtual void func(Common::U32String *msg = nullptr);
  void client1() { func(); }
  void client2() { Common::U32String msg; func(&msg); }
};
struct Derived1 : Base {
  void func(Common::U32String *msg = nullptr) override
  {
    // do something with msg
  }
};

struct Derived2 : Derived1 {
  void func(Common::U32String *msg = nullptr) override
  {
    // Do something
    Derived1::func(); // <-- Oops, missing msg
  }
};
```

Reply:
I understand, but it never happened and should not happen in our case. The superclass just returns `false`.